sanic/config.py

```diff
@@ -137,9 +137,21 @@ def __setitem__(self, attr, value) -> None:
         self.update({attr: value})
 
     def update(self, *other, **kwargs) -> None:
-        other_mapping = {k: v for item in other for k, v in dict(item).items()}
-        super().update(*other, **kwargs)
-        for attr, value in {**other_mapping, **kwargs}.items():
+        kwargs.update({k: v for item in other for k, v in dict(item).items()})
+        setters: Dict[str, Any] = {
+            k: kwargs.pop(k)
+            for k in {**kwargs}.keys()
```

Comment:
Should work either way, I believe, but just to simplify a bit.
```suggestion
            for k in kwargs
```

Reply:
We need a copy because we are changing the values of the dict in the iteration. I believe we would get an exception otherwise.

Reply:
It does indeed!